packages/fluentui/react-northstar/src/components/Carousel/Carousel.tsx

```diff
@@ -297,29 +332,29 @@ export const Carousel = (React.forwardRef<HTMLDivElement, CarouselProps>((props,
             items.map((item, index) => {
               const itemRef = itemRefs[index];
               const active = activeIndex === index;
-              let slideToNext = prevActiveIndex < activeIndex;
-              const initialMounting = prevActiveIndex === -1;
+              const initialMounting = typeof dir === 'undefined';
+
+              let animationName = '';
+              if (!initialMounting) {
+                if (!active) {
+                  if (dir === 'right') {
+                    animationName = animationExitToLeft;
+                  } else {
+                    animationName = animationExitToRight;
+                  }
```

Comment:
nit: maybe a ternary is more compact ?

```suggestion
                  animationName = dir === 'right ? animationExitToLeft : animationExitToRight
```